generators/app/templates/scripts/_karma.conf.js

```diff
@@ -1,62 +1,108 @@
-/* eslint-disable no-console */
+/* eslint-disable no-console, camelcase */
 const serveStatic = require('serve-static');
-const path = require('path');
-const serve = serveStatic(
-  path.join(__dirname, '..'),
-  {index: ['index.html', 'index.htm']}
-);
+const pkg = require('../package.json');
 
+/* allow static files to be served, note karma will takeover the /test directory */
 const StaticMiddlewareFactory = function(config) {
-  console.log(`**** Dev server started at http://${config.listenAddress}:${config.port}/ *****`);
```

Comment:
is this info being printed elsewhere?

Reply:
The karma server prints it when it starts up. Oftentimes the address we were printing here is the address we were passed. This means that when port `9999` is in use and get incremented by karma to `10000`. we still print `9999`.

Reply:
cool.